mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -118,8 +118,16 @@ The new annotations model is similar to the new labels model and will be applied
 
 There are now two annotation-related fields with the new model, the `annotations` and the output-only `effective_annotations` fields.
 
+### Provider default values shown at plan-time
+
+`project`, `region`, and `zone` fields will now display their values during plan-time instead of defaulting to the unknown `(known after apply)` value normally displayed for `computed` fields. These values will be taken from either the Terraform resource config file, provider config, or local environment variables, depending on which variables are supplied by the user, matching the existing per-resource functionality for what default values are used in execution of a Terraform plan.
```

Comment:
```suggestion
`project`, `region`, and `zone` fields will now display their values during plan-time instead of defaulting to the unknown `(known after apply)` value normally displayed for [Computed](https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors#computed) fields. These values will be taken from either the Terraform resource config file, provider config, or local environment variables, depending on which variables are supplied by the user, matching the existing per-resource functionality for what default values are used in execution of a Terraform plan.
```
Not sure if adding a link to the docs for what "Computed" means will be more or less confusing for users. I'll defer to your opinion.

Reply:
I'm not sure the level of expected technical specification in here -- if erring on the side of basic terminology, I'll rewrite it to describe it along the lines of "fields without fixed default values"

Reply:
That sounds even better. Thanks!

Reply:
The new wording is in, sounds good?

Reply:
added another suggestion on second thought